modules/devshell.nix

```diff
@@ -216,6 +216,14 @@ in
       '';
     };
 
+    meta = mkOption {
+      type = types.attrsOf types.any;
```

Comment:
Do we already have better emergent typing of the `meta` attribute? Does anybody know?